.github/workflows/build.yaml

```diff
@@ -18,6 +18,10 @@ jobs:
         - '^(7\.3)-apache$'
         - '^(7\.4)-apache$'
         - '^(8\.0)-apache$'
+        - '^(7\.2)-fpm'

```

Comment:
php 7.2 is eol en php 7.3 is over 4 maanden EOL, lijkt het me niet logisch om daar images voor te releasen, is enkel 8.0 en eventueel 7.4 indien nodig niet genoeg?